src/provider/systemdmanager/manager.rs

```diff
@@ -369,7 +390,6 @@ impl SystemdManager {
     // Check if the unit name is valid and append .service if needed
     // Cannot currently fail, I'll need to dig into what is a valid unit
     // name before adding checks
-    #[allow(clippy::unnecessary_wraps)]
```

Comment:
Why? Clippy still complains

Reply:
For me it started complaining about a non-existent #allow target at some point ..